src/DeFiScripts.sol

```diff
@@ -326,3 +327,16 @@ contract ApproveAndSwap {
         IERC20(sellToken).forceApprove(to, 0);
     }
 }
+
+contract CCTPBridgeActions {
```

Comment:
I think this should be in its own file since it's not really related to DeFi.

Reply:
That's true..
Will make it into separate file then 👍